src/main/java/org/spongepowered/common/command/WrapperICommandSender.java

```diff
@@ -119,7 +120,7 @@ public MinecraftServer getServer() {
         if (world != null) {
             return world.getMinecraftServer();
         }
-        return null;
+        return SpongeImpl.getServer();
```

Comment:
Agreement on whether this should be done? If this returns null then vanilla commands cannot get the `CommandHandler` for the world and so won't work as intended. This is to provide a default server to execute the command on if a located source isn't supplied.